packages/docs/src/examples/Transaction.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react'
+import ReactDOM from 'react-dom'
+import { parseEther } from '@ethersproject/units'
+import { Mainnet, DAppProvider, useSendTransaction, useEthers, Config } from '@usedapp/core'
+import { ChainId } from '../../../core/src/constants/chainId'
+
+const config: Config = {
+    readOnlyChainId: Mainnet.chainId,
+    readOnlyUrls: {
+      [Mainnet.chainId]: 'https://mainnet.infura.io/v3/57fc2c19095745e59ab96a4aa87dada8',
+    },
+}
+
+ReactDOM.render(
+    <DAppProvider config={config}>
+      <TokenBalance />
+    </DAppProvider>,
+    document.getElementById('root')
+)
+
+export function TokenBalance() {
+    const { chainId } = useEthers()
+    const { sendTransaction, state } = useSendTransaction()
+    
+    const status = state.status
+    const disabled = chainId === ChainId.Mainnet
+    const amount = '0.5'
+    const address = '0xe13610d0a3e4303c70791773C5DF8Bb16de185d1'
+
+    const send = () => {
+        sendTransaction({ to: address, value: parseEther(amount) })
+    }
+
+    const { activateBrowserWallet, account } = useEthers()
+
+    return (
+    <div>
+        <button onClick={() => activateBrowserWallet()}>Connect</button>
```

Comment:
Shall we make the button disappear when we have already connected?
(Applies to every other example as well)

Reply:
I'll make task for that and i can fix that, also from me is to export all configs except for getting started to the other file. What do You think?

Reply:
We can give it a try.